### PR TITLE
[Resolves  #54] Add pre-commit and contributing guidelines

### LINKED ISSUE #54: [DOCS] Missing contributing docs
There is no documentation about contribution.

### PR DESCRIPTION
**Pull request by felipeffm**
This pull request has added contributing guidelines to documentation and added pre-commit with some hooks.

### CODE REVIEW

File: .pre-commit-config.yaml [hunk 1]
```diff
@@ -0,0 +1,90 @@
+exclude: "^(docs|.devcontainer|.venv|.vscode|templates|.*tests.*snapshots.*).*|.*[.]pyi"
+default_language_version:
+  ruby: 2.7.2
```
Comment (felipeangelimvieira) [inline .pre-commit-config.yaml:3]: Felipe, maybe here should be python 3.10?

**Review by felipeangelimvieira — COMMENTED** [review]
Awesome Felipe, I believe that just the default language should be update and it's ready

File: .pre-commit-config.yaml (hunk 1 above)
Comment (felipeangelimvieira) [inline .pre-commit-config.yaml:3]: Felipe, maybe here should be python 3.10?

**Review by felipeangelimvieira — CHANGES_REQUESTED** [review]

File: .pre-commit-config.yaml (hunk 1 above)
Comment (felipeffm) [reply .pre-commit-config.yaml]: The default language does not need to be specified so I deleted the line.

**Review by felipeangelimvieira — APPROVED** [review]

**Pull request merged**